packages/grid/_modules_/grid/hooks/features/filter/useGridFilter.ts

```diff
@@ -36,7 +36,6 @@ const checkFilterModelValidity = (model: GridFilterModel) => {
  * @requires useGridColumns (state, method, event)
  * @requires useGridParamsApi (method)
  * @requires useGridRows (event)
- * @requires useGridControlState (method)
```

Comment:
Do they require `useGridStateInitialization (method)` ?

Reply:
I remove `useGridControlState` from the `@requires` because I take for granted that any hook is after `useGridInitialization` so any hook inside `useGridInitialization` is available.

That way we only keep the list of the deps handled directly in `useDataGrid(Pro)Component` and which could be moved for another reason.